packages/host/app/services/store.ts

```diff
@@ -219,6 +220,24 @@ export default class StoreService extends Service {
     return { url, card, error };
   }
 
+  // This method is used for specific scenarios where you just want an instance
+  // that is _not_ part of the identity map and detached from the store. This
+  // may include things like tests, freestyle guide, perhaps you just need a
+  // couple attributes from an instance and that's it, etc.
+  async getInstanceDetachedFromStore(
+    url: string,
+  ): Promise<CardDef | undefined> {
+    let doc = await this.cardService.fetchJSON(url);
+    if (!doc) {
+      return undefined;
+    }
+    return await this.cardService.createFromSerialized(
+      doc.data as LooseCardResource,
+      doc,
+      new URL(url),
+    );
+  }
+
```

Comment:
This is a low level method that consumer can use to get a card instance that is NOT part of the store.